File: UI/eturistickaagencija_mobile/lib/model/Uplata.dart
import 'dart:convert';

class Uplata {
  late int? korisnikId;
  final double iznos;
  final String brojTransakcije;
  final DateTime datumTransakcije;

  Uplata(
      {this.korisnikId,
      required this.iznos,
      required this.brojTransakcije,
      required this.datumTransakcije});

  Map<String, dynamic> toJson() => {
        "korisnikId": korisnikId,
        "iznos": iznos,
        "brojTransakcije": brojTransakcije,
        "datumTransakcije": datumTransakcije.toString(),
      };
}

File: eTuristickaAgencija.Models/Uplata.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTuristickaAgencija.Models
{
    public class Uplata
    {
        public int Id { get; set; }
        public decimal Iznos { get; set; }
        public string BrojTransakcije { get; set; }
        public DateTime DatumTransakcije { get; set; }
        public int KorisnikId { get; set; }
    }
}
